imagemagick-stable/snapcraft.yaml

```diff
@@ -1,5 +1,10 @@
+<<<<<<< HEAD:imagemagick-edge/snapcraft.yaml
+name: imagemagick-edge
+version: 7.0.2~git
+=======
 name: imagemagick-stable
 version: 6.9.7
+>>>>>>> ddb719ede8d95bd9e1362f9dd91033c1581ffb5c:imagemagick-stable/snapcraft.yaml
 summary: ImageMagick is a software suite to create, edit, or convert bitmap images.
```

Comment:
You've uncovered the stuff I was rushing through. Made some errors `git rm`ing, and just `rm`ed and messing up tidying